docs/diagnostics/OT1000.md

```diff
@@ -0,0 +1,42 @@
+# OpenTelemetry .NET Diagnostic: OT1000
```

Comment:
Could this be renamed to `OTEL1000`?

https://github.com/open-telemetry/opentelemetry-specification/tree/7b6de1f7dc2b40e22ab747b7810762222c348104/specification#project-naming

> Refrain from using "OT" in order to avoid confusion with the now deprecated "OpenTracing" project.

Reply:
@sharwell Random Q. Are there any guidelines for choosing diagnostic ids? I think most (all?) I have ever seen are DDNNNN any reason for that?

Reply:
@terrajobst would you have insight here for recommended use of ExperimentalAttribute?

Reply:
I don't have any great insights into choosing diagnostic IDs except for a few things:

* It's a source breaking change to change them (e.g. existing suppressions are now ignored)
* The C# compiler requires them to be legal identifiers
* You want to include a number
* You want to choose a specific prefix that is specific to your project
* You want to something short (say, less than 15 characters)

I don't think you want to limit your prefix to two-characters (CSXXX and CAXXXX) but rather something longer to avoid conflicts. For the System.* diagnostics we use `SYSLIB` for instance. `OTEL` seems quite reasonable.

If you already know of another project using the `OT` prefix I'd highly recommend to go with something else or consumers that use both will be in trouble.

Please note that we generally want to enable people using IDs in a centralized location (e.g. an `editor.config` file or a project file for global suppressions) so don't assume that conflicts only arise for statements that use both, therefore you generally want globally unique identifiers for diagnostics.

Does this help?

Reply:
Thanks @terrajobst! I updated to `OTEL1000` & `OTEL1001`.

Reply:
FYI, we added this to our [documentation too](https://learn.microsoft.com/en-us/dotnet/csharp/roslyn-sdk/choosing-diagnostic-ids).

Reply:
Thanks @terrajobst!